packages/api/src/controllers/asset.ts

```diff
@@ -74,18 +77,32 @@ function isPrivatePlaybackPolicy(playbackPolicy: PlaybackPolicy) {
   return true;
 }
 
-function defaultObjectStoreId(
-  { config, body }: Request,
+const secondaryStorageExperiment = "secondary_vod_storage";
```

Comment:
Super nit: I think the experiments were designed to use in the URL, so maybe `secondaryVodStorage` or `secondary-vod-storage`?